client/gg_client.cpp

```diff
@@ -364,7 +387,7 @@ int GG::Client::stream(std::string token) {
 
 	// create colour lookup table for generating ascii colour codes 
 	CharFormat* colour_lookup_table = new CharFormat[16 * 16 * 16];
-    this->_populate_color_lookup_table(colour_lookup_table, ColorSchemeKind::MSE);
+  this->_populate_color_lookup_table(colour_lookup_table, ColorSchemeKind::MSE, NUM_XTERM_COLOURS, false);
```

Comment:
kewl, so should there be some new "num colors" variable kept in the user session, that could be passed into this function to configure output?

++, how do you configure whether the background is colored or not?

Reply:
yeah! the two parameters we'd want to vary are `num_colours` and `disable_bg_colours`. Both work separately, so if you wanted to stop the background from being coloured, you'd pass true to disable background colours.

Reply:
ohhh I see, cool! This should be easy to embed into the sessions then; thanks!!!